src/lib/web-component/log-view/log-view.css

```diff
@@ -19,7 +24,11 @@
 }
 
 .log-details {
-  margin: 50px 15px 10px;
+  margin: 40px 0 10px;
+  white-space: pre-wrap;
+  max-height: 450px;
```

Comment:
Setting the `max-height` to 450px since I used 50px pixels margin in total at [line 27](https://github.com/mozilla/extension-activity-monitor/pull/15/files#diff-3e34fcdde14c53a51242dcc1edf18813R27) (40px top and 10px bottom) to give it box like look.